src/messages.js

```diff
@@ -0,0 +1,9 @@
+var req = require.context('../lang', true, /\.json.*$/);  
+var exports = {};
+
+req.keys().forEach(function (file) {
+  var locale = file.replace('./', '').replace('.json', '');
+  exports[locale] = req(file);
+});
```

Comment:
This is taking advantage of the way Webpack parses require statements, I'll add some comments. It just goes through each file and uses the file name as a module export.

It's all here, check require.context:
https://webpack.github.io/docs/context.html